code/script.js

```diff
@@ -1,47 +1,160 @@
-      <section class="user-msg">
-        <div class="bubble user-bubble">
-          <p>${message}</p>
-        </div>
-        <img src="assets/user.png" alt="User" />  
-      </section>
-    `
-  } else if (sender === 'bot') {
-    chat.innerHTML += `
-      <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
-        <div class="bubble bot-bubble">
-          <p>${message}</p>
-        </div>
-      </section>
-    `
-  }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
-}
-
-// Starts here
-const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
-
-// Set up your eventlisteners here
-
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000)
+// All the DOM selectors stored as short variables
+const chat = document.getElementById("chat");
+const inputWrapper = document.getElementById("input-wrapper");
+const nameInput = document.getElementById("name-input");
+const form = document.getElementById("name-form");
+
+// const sendBtn = document.getElementById("send-btn")
+const audio = new Audio("drill.mp3");
+
+// Global variables, if you need any, declared here
+let questionStep = 1;
+
+// Function to display the message on the screen both for bot and user
+const botAnswer = (inputMessage) => {
+  showMessage(inputMessage, "bot");
+};
+const userAnswer = (inputMessage) => {
+  showMessage(inputMessage, "user");
+};
+
+// Question flow functions
+const handleInput = (message) => {
+  if (questionStep === 2) {
+    userAnswer(message);
+    setTimeout(() => moreServices(message), 2000);
+  } else if (questionStep === 3) {
+    userAnswer(message);
+    setTimeout(() => priceInformation(message), 1000);
+  } else {
+    userAnswer(message);
+    setTimeout(goodBye, 1000);
+  }
+};
+// This function will add a chat bubble in the correct place based on who the sender is
+const showMessage = (message, sender) => {
+  if (sender === "user") {
+    chat.innerHTML += `
+      <section class="user-msg">
+        <div class="bubble user-bubble">
+          <p>${message}</p>
+        </div>
+        <img src="assets/user.png" alt="User" />  
+      </section>
+    `;
+  } else if (sender === "bot") {
+    console.log("This works");
+    chat.innerHTML += `
+      <section class="bot-msg">
+        <img src="assets/bot.gif" alt="Bot" />
+        <div class="bubble bot-bubble">
+          <p>${message}</p>
+        </div>
+      </section>
+    `;
+  }
+
+  // Audio
+
+  audio.currentTime = 0;
+  audio.play();

```

Comment:
I see error message in console regarding this line, I am not sure what is the issue though!